### DISCUSSION TITLE
how to send the last row created data to a webhook?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2210] [Help] [answered]
Why this doesn't work
![image](https://user-images.githubusercontent.com/3023526/127861560-58a60e83-e13a-4343-b2b8-7e6dac7f95c8.png)
and this works 
![image](https://user-images.githubusercontent.com/3023526/127861834-d0899dab-58ee-451e-8dfc-258dbd666c40.png)
?

**Comment by mjashanks** [comment, marked as answer]
Hey @oswaldB, try using `{{ JSONstringify trigger.row }}`.

There's some discussion around this in here: https://github.com/Budibase/budibase/discussions/2016

**Reply by oswaldB** [reply]
Thank you!
![image](https://user-images.githubusercontent.com/3023526/127863760-05ac88c8-a0c5-4670-8c82-33b65ad9ddd0.png)
this works fine.
May you can add the JSONstringify into the helper?
